pkg/client/multi.go

```diff
@@ -40,6 +40,22 @@ func aggregate[T any, V ObjectPointer[T]](ctx context.Context, factory ProjectCl
 	return aggregateOptionalNaming[T, V](ctx, true, factory, cb)
 }
 
+// parseProject accepts an argument name that is either of the form resource or project::resource, and ClientFactory
+// If the name is not of project::resource form, it returns defaultProject, name, nil
+// otherwise, it parses off the project name and returns project, name without project::, nil
+func parseProject(ctx context.Context, name string, factory ProjectClientFactory) (pc Client, parsedArg string, err error) {
```

Comment:
nit: Slight rewording to make it easier to follow
```suggestion
// parseProject accepts a name string (in the form of resource or project::resource) and a ClientFactory.
// If the name string is not of project::resource form, it returns a client for the default project, the name, and
// nil. Otherwise, it parses off the project name and returns a client for that project, the name without project::, 
// and nil
func parseProject(ctx context.Context, name string, factory ProjectClientFactory) (pc Client, parsedArg string, err error) {
```